pages/learn/foundations/about-nextjs/index.mdx

```diff
@@ -1,6 +1,28 @@
-import NotTranslateYet from '@components/NotTranslateYet';
 import LearnLayout from '@components/LearnLayout';
 
-<NotTranslateYet/>
+# 前情提要
+
+熟悉 JavaScript、React 及相關的網頁開發概念，能夠有效率的使用Next.js。可是Javascript和React是一個廣大的議題，要如何得知自己有能力開始學習Next.js?
+
+歡迎來到Next.js基礎課程! 這個課程對於新手非常友好且提供許多範例演練，在這裡我們將會一步一步帶著你了解學習Next.js之前所需要的相關知識。首先，利用 Javascript 一步步建立一個簡單的應用程式，完成之後，再把它轉變成React、Next.js的形式。
+
+每個區塊的內容將延伸上一個區塊討論的議題，這讓你可以依據你目前的程度決定從哪裡開始。
+
+<blockquote className="px-4 py-1 my-10 bg-[#f7f7f7]">
+  此教學建立於你已經對於HTML, CSS, JavaScript有一定的了解，但沒有React相關知識。所以，如果你已經對於React相當熟悉，請至
+  <a className="text-[#0070f3]" target="_blank" href="/learn/foundations/from-react-to-nextjs/getting-started-with-nextjs">快速開始Next.js</a>
+  或
+  <a className="text-[#0070f3]" target="_blank" href="/learn/basics/create-nextjs-app">打造你的第一個Next.js應用程式</a>。
+</blockquote>
+
+<b className="block">加入我們的社群</b>
+
+如果你對於Next.js或此課程有任何問題，歡迎到我們的<a className="text-[#0070f3]" href="https://discord.gg/Q3AsD4efFC">Discord社群</a>提問。
+
+那我們開始吧!
+
+<a className="inline-block py-2.5 px-8 mt-8 text-white text-center bg-[#0070f3] rounded-lg cursor-pointer shadow-[rgba(0,118,255,.39)] shadow-lg hover:opacity-90">前往課程 →</a>
```

Comment:
I think we should make a `<Button />` component and use in MDX.

Reply:
You got it.